python/aitemplate/compiler/transform/fuse_utils.py

```diff
@@ -87,6 +87,11 @@ def _find_fusion_root(tensor: Tensor, fusion_patterns: List[Any]) -> int:
             dst_op_tensor = dst_op._attrs["outputs"]
             if len(dst_op_tensor) != 1:
                 break
+            if dst_op_tensor[0]._attrs["is_output"]:
```

Comment:
are these changes expected?

Reply:
No, this was exported from other diff. Let me disconnect this one and re-export.